sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -115,113 +115,115 @@ The following section provides several code snippets using the [above created](#
 ### Create a Key
 Create a Key to be stored in the Azure Key Vault. If a key with the same name already exists, then a new version of the key is created.
 
-```C# CreateKey
-// Create a key. Note that you can specify the type of key
-// i.e. Elliptic curve, Hardware Elliptic Curve, RSA
-Key key = client.CreateKey("key-name", KeyType.Rsa);
-
-Console.WriteLine(key.Name);
-Console.WriteLine(key.KeyMaterial.KeyType);
-
-// Create a software RSA key
-var rsaCreateKey = new RsaKeyCreateOptions("rsa-key-name", hsm: false);
-Key rsaKey = client.CreateRsaKey(rsaCreateKey);
-
-Console.WriteLine(rsaKey.Name);
-Console.WriteLine(rsaKey.KeyMaterial.KeyType);
-
-// Create a hardware Elliptic Curve key
-var echsmkey = new EcKeyCreateOptions("ec-key-name", hsm: true);
-Key ecKey = client.CreateEcKey(echsmkey);
-
-Console.WriteLine(ecKey.Name);
-Console.WriteLine(ecKey.KeyMaterial.KeyType);
+```C# CreateKey
+// Create a key. Note that you can specify the type of key
+// i.e. Elliptic curve, Hardware Elliptic Curve, RSA
+Key key = client.CreateKey("key-name", KeyType.Rsa);
+
+Console.WriteLine(key.Name);
+Console.WriteLine(key.KeyMaterial.KeyType);
+
+// Create a software RSA key
+var rsaCreateKey = new RsaKeyCreateOptions("rsa-key-name", hsm: false);
```

Comment:
Do not modify the code in the README directly. Instead, modify the code in associated samples/SampleSnippets.cs that has #regions with the same name as the text here after "c#", then run a ps1 script in the root "eng" or "tools" folder, something like Update-Snippets.ps1.